sa/database.go

```diff
@@ -142,4 +142,6 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.OCSPResponse{}, "ocspResponses").SetKeys(true, "ID")
 	dbMap.AddTableWithName(core.CRL{}, "crls").SetKeys(false, "Serial")
 	dbMap.AddTableWithName(core.DeniedCSR{}, "deniedCSRs").SetKeys(true, "ID")
+	sctTable := dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID")
+	sctTable.SetUniqueTogether("LogID", "CertificateSerial").SetVersionCol("LockCol")
```

Comment:
I think this is unnecessary now that we are using Goose for migrations, right? This uniqueness constraint is expressed in the .sql file.